src/pages/account-ownership-transfer/[guid].tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from 'react'
+import LoginRequired from 'components/login-required'
+import {useRouter} from 'next/router'
+import toast from 'react-hot-toast'
+import axios from 'axios'
+import {
+  getAuthorizationHeader,
+  getTokenFromCookieHeaders,
+  AUTH_DOMAIN,
+} from 'utils/auth'
+
+async function confirmAccountOwnershipTransfer(guid: string) {
+  let transferSucceeded
```

Comment:
style[nbd]: for most of this codebase, we avoid `let` entirely and leverage `const` with multiple returns. This isn't critical, but it's a style drift so I'm bringing it up.

is using `let` and sticking to a single return value a general approach you prefer?

Reply:
I default to using `const` 99% of the time. If I end up using `let`, it is generally because I'm finding myself needing a value at a higher level of scope than where it will be assigned.

Looking back at this function, the `let` isn't doing much for me here. I would prefer multiple returns in this case. I seem to remember that there was a lot more going on in this method and that the `let` is the artifact of a simplifying refactoring.

In Ruby code I prefer a single return over multiple returns if it doesn't make the code awkward. I find that makes the flow and logic easier to reason about. Especially in a language with implicit returns. I bet that preference bleeds into the JS that I write without even thinking about it.